Commonvars.mk

```diff
@@ -54,7 +54,8 @@ EXTRA_TAG ?= -dev
 PLUGIN_TAG := $(VERSION_TAG)$(EXTRA_TAG)
 
 # plugin name - used as a base for full plugin name and container for extracting rootfs
-PLUGIN_NAME=$(DOCKER_HUB_REPO)/docker-volume-vsphere
+PLUGIN_NAME ?= $(DOCKER_HUB_REPO)/docker-volume-vsphere
```

Comment:
?= indicates to set the variable only if it's not set/doesn't have a value.
This change is because of we will recursively call the Makefile for the shared plugin, to avoid duplicate Makefile code.
See Line 67 in the new plugin_dockerbuild/Makefile:
```
shared-clean:
	PLUGIN_NAME=$(SHARED_PLUGIN_NAME) BINARY=$(SHARED_BINARY) $(MAKE) clean

```